CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contributing guidelines
+
+## Very important notice about the Javalib
+
+If you haven't read it, ***read the very important notice about the Javalib
+in the [Developer documentation](./DEVELOPING.md)***.
+
+## General workflow
+
+This the general workflow for contributing to Scala.js.
+
+1.  Make sure you have signed the
+    [Scala CLA](http://typesafe.com/contribute/cla/scala).
+    If not, sign it.
+2.  You should always perform your work in its own Git branch.
+    The branch should be given a descriptive name that explains its intent.
+3.  When the feature or fix is completed you should open a
+    [Pull Request](https://help.github.com/articles/using-pull-requests) on GitHub.
+4.  The Pull Request should be reviewed by other maintainers (as many as feasible/practical),
+    among which at least one core developer.
+    Independent contributors can also participate in the review process,
+    and are encouraged to do so.
+5.  After the review, you should resolve issues brought up by the reviewers as needed
+    (amending or adding commits to address reviewers' comments), iterating until
+    the reviewers give their thumbs up, the "LGTM" (acronym for "Looks Good To Me").
+6.  Once the code has passed review the Pull Request can be merged into the distribution.
+
+## Pull Request Requirements
+
+In order for a Pull Request to be considered, it has to meet these requirements:
+
+1.  Live up to the current code standard:
+    - Not violate [DRY](http://programmer.97things.oreilly.com/wiki/index.php/Don%27t_Repeat_Yourself).
+    - [Boy Scout Rule](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule) should be applied.
+2.  Be accompanied by appropriate tests.
+
+If *all* of these requirements are not met then the code should **not** be
```

Comment:
If not *all* of these requirements... (this actually does not mean what you want it to)